monkey/monkey_island/cc/services/authentication_service/flask_resources/__init__.py

```diff
@@ -5,3 +5,4 @@
 from .register_resources import register_resources
 from .agent_otp import AgentOTP
 from .agent_otp_login import AgentOTPLogin
+from .token import Token
```

Comment:
Wait, what's the purpose of these imports if all resources are registered through `register_resources`?